crypto/falconWrapper.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2019-2021 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package crypto
+
+import (
+	cfalcon "github.com/algorand/falcon"
+)
+
+const (
+	// FalconSeedSize Represents the size in bytes of the random bytes used to generate Falcon keys
+	FalconSeedSize = 48
+)
+
+type (
+	// FPublicKey is a wrapper for cfalcon.PublicKeySizey (used for packing)
+	FPublicKey [cfalcon.PublicKeySize]byte
+	// FSecretKey is a wrapper for cfalcon.PrivateKeySize (used for packing)
+	FSecretKey [cfalcon.PrivateKeySize]byte
+	// FalconSeed represents the seed which is being used to generate Falcon keys
+	FalconSeed [FalconSeedSize]byte
+)
+
+// FalconSigner is the implementation of Signer for the Falcon signature scheme.
+type FalconSigner struct {
+	_struct struct{} `codec:",omitempty,omitemptyarray"`
+
+	PublicKey FPublicKey `codec:"pk"`
```

Comment:
add allocbound

Reply:
FPublicKey is a fixed-length slice. do we need to bound this even if it is fixed?